src/legacy/ui/public/vislib/lib/types/point_series.js

```diff
@@ -203,11 +207,18 @@ export const vislibPointSeriesTypes = {
 
   heatmap: (cfg, data) => {
     const defaults = create()(cfg, data);
-    const seriesLimit = 25;
+    const seriesLimit = config.get('visualization:heatmap:maxBuckets');
```

Comment:
rather than importing `chrome`, gettng the `uiSettingsClient` and reading out the setting, can we rather do this on the top level (vislib.js or vis.js) and pass it in ? (we are doing this as part of np migration process, i would prefer to not generate new tech depth :) )

Reply:
done